packages/create-block/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Changes
```

Comment:
I was thinking about that, and it seems to me like it should be classified as an `Enhancement`. My reasoning is that it doesn't impact existing projects, but instead changes how the newly scaffolded project is configured.